lib/src/flutterflow_api_client.dart

```diff
@@ -135,25 +133,28 @@ Future<String?> exportCode({
 
 // Extract files to the specified directory without a project-named
 // parent folder.
-void extractArchiveToCurrentDirectory(
-  Archive projectFolder,
-  String destinationPath,
-) {
+void extractArchiveTo(
+    Archive projectFolder, String destinationPath, bool unzipToParentFolder) {
+  final ignore = FlutterFlowIgnore(path: destinationPath);
+
   for (final file in projectFolder.files) {
     if (file.isFile) {
-      final data = file.content as List<int>;
-      final filename = file.name;
+      final relativeFilename =
+          path_util.joinAll(path_util.split(file.name).sublist(1));
+
+      // Found on .flutterflowignore, move on.
+      if (ignore.matches(relativeFilename)) {
+        print('Ignoring $relativeFilename, file remained unchanged.');
```

Comment:
Nit: logs like this usually go to `stderr` (it's debatable of course).

Reply:
Makes sense, the rest of the code is like this.